Building with cmake (version 3.10.2) command leads to build error

Building with cmake (version 3.10.2) command leads to build error as shown

CMake Error at v7/CMakeLists.txt:38 (target_link_libraries):
  Object library target "bplib_v7" may not link to anything.

--- a/v7/CMakeLists.txt
+++ b/v7/CMakeLists.txt
@@ -35,8 +35,6 @@ target_include_directories(bplib_v7 PUBLIC
      ${CMAKE_CURRENT_SOURCE_DIR}/inc
 )
 
-target_link_libraries(bplib_v7 ${TINYCBOR_LDFLAGS})
-
 # compile this submodule as c99
 target_compile_features(bplib_v7 PRIVATE c_std_99)
 

